src/util/XDRCereal.h

```diff
@@ -118,14 +138,7 @@ xdr_to_string(const T& t, std::string const& name = "", bool compact = false)
         cereal::JSONOutputArchive ar(
             os, compact ? cereal::JSONOutputArchive::Options::NoIndent()
                         : cereal::JSONOutputArchive::Options::Default());
-        if (!name.empty())
-        {
-            ar(cereal::make_nvp(name, t));
-        }
-        else
-        {
-            ar(t);
-        }
+        ar(cereal::make_nvp(name, t));
```

Comment:
This doesn't seem quite right to me (and I don't think it was right before either). I think this should be done with `xdr::archive` to make sure it uses the appropriate archive adapter, thereby presenting access to `cereal_override`.

Reply:
Fixed! Thanks for pointing this out. I never noticed it.